modules/geochemistry/include/utils/GeochemistryConstants.h

```diff
@@ -0,0 +1,18 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#pragma once
+
+#include "MooseTypes.h"
+
+constexpr Real MOLES_PER_KG_WATER = 55.51;
+constexpr Real LOGTEN = 2.30258509299404;
+constexpr Real FARADAY = 96485.3415;    // Coulombs . mol^-1
+constexpr Real GAS_CONSTANT = 8.314472; // m^2 . kg . s^-2 . K^-1 . mol^-1
+constexpr Real CELSIUS_TO_KELVIN = 273.15;
```

Comment:
This should all be in a namespace. 

Also: you're sneaky here putting in constants you haven't used yet 😬

Reply:
Heheheh, yea, those constants are used in my non-pushed stuff.  Hopefully you don't mind.  I'll make the namespace in  few mins.